datus/utils/sql_utils.py

```
@@ -822,3 +822,19 @@ def normalize_sql(sql: str) -> str:
+def format_sql_to_pretty(sql: str, dialect: str) -> str:
+    if not sql:
+        return sql
+    read_dialect = parse_read_dialect(dialect)
+    """Pretty print SQL if possible, otherwise return the original text."""
+    if not sql:
+        return ""
+    try:
+        formatted = sqlglot.transpile(sql, read=read_dialect, pretty=True)
+        if formatted:
+            return formatted[0]
+    except Exception as exc:
```

Ruff (0.14.6)

Do not catch blind exception: `Exception`

(BLE001)